views/layouts/default.html.erb

```diff
@@ -29,13 +29,14 @@
             <nav class="pd-b">
               <a href="/" title="Welcome">Welcome</a>
               <a href="/#sec-hardfork" title="Bigger&nbsp;blocks">Bigger&nbsp;blocks</a>
+              <a href="/onboard.html" title="Who's Onboard">Who's&nbsp;Onboard</a>
```

Comment:
"Patches" could be moved somewhere else, like in FAQ. It's rather technical.

Small screens are pretty rare, anyway. We live in the era of 1920 being the default monitor width. I don't think it's such a big issue.